src/get-started/test-service.md

```diff
@@ -289,27 +288,26 @@ impl CryptocurrencyApi {
 }
 ```
 
-`create_wallet` returns a secret key along with the created transaction
+`create_wallet` returns a key pair along with the created transaction
 because it may be needed to sign other transactions authorized by the wallet
 owner.
 
-Inside, all wrapper methods call the `inner` API instance; for example,
-`get_wallet` is implemented as:
+Inside, all wrapper methods invoke methods of the `inner` API instance;
+for example, `get_wallet` is implemented as:
 
 ```rust
 fn get_wallet(&self, pub_key: PublicKey) -> Wallet {
     self.inner
-        .public(ApiKind::Service("cryptocurrency"))
+        .public(ApiKind::Service(INSTANCE_NAME))
         .query(&WalletQuery { pub_key })
         .get("v1/wallet")
         .unwrap()
 }
 ```
 
 That is, the method performs an HTTP GET request with the URL address
-corresponding to a service
-[with the specified name](../architecture/services.md#service-identifiers)
-and a `v1/wallet/…` path within the service API. When we created the service,
+corresponding to a service with the specified name and a `v1/wallet/…`
```

Comment:
This was an error previously, but `…` in the path is bogus; the path should be simply `v1/wallet`.